doc/manual/manual.tex

```diff
@@ -2005,13 +2005,13 @@ \subsubsection{Combined formulations}
 (Section~\ref{sec:tala}).
 
 \item
-``boussinesq approximation'': This formulation sets the mass conservation approximation to ``incompressible'', 
+``Boussinesq approximation'': This formulation sets the mass conservation approximation to ``incompressible'', 
```

Comment:
this occurrence of the word needs to match what is being used in the .prm files. Did you change that as well?